components/Tabs/src/MDCTabBarViewController.m

```diff
@@ -241,5 +242,14 @@ - (UIBarPosition)positionForBar:(id <UIBarPositioning>)bar {
   }
 }
 
+#pragma mark - UIViewController status bar
+
+- (nullable UIViewController *)childViewControllerForStatusBarStyle {
+  return _selectedViewController;
+}
+
+- (nullable UIViewController *)childViewControllerForStatusBarHidden {
+  return _selectedViewController;
+}
 
```

Comment:
Maybe also `preferredStatusBarUpdateAnimation`?

Reply:
I think the default value is OK, but perhaps it could be overridden in a separate PR?

Reply:
Yeah I don't know if it's really necessary. I just see that it's done in some similar code I have open here, but that could be historical noise. 

Sounds good, let's just leave it to a separate PR if it seems to be necessary later.